apps/protocol-frontend/src/components/CreateDaoLayout.tsx

```diff
@@ -4,18 +4,16 @@ import PageHeading from './PageHeading';
 import DaoTextareaForm from './DaoTextareaForm';
 import DaoImportCard from './DaoImportCard';
 import ModalWrapper from './ModalWrapper';
-import ImportHistoryTable from './ImportHistoryTable';
```

Comment:
(arbitrary thread)

We should probably redirect somewhere when a dao is created. Especially if there is no table.

Reply:
It may even make sense to move the modal directly to the create dao button in the profile

Reply:
Good idea! Instead of linking to it from there, we should pop that open itself since it is a modal

Reply:
On the redirect: I agree. I think once the context refactor is in we should redirect to the newly created DAO page. I held off on that since the user would need to refresh that page for them to be considered a member, but if we handle that via invalidation we can direct to that new page.

Reply:
One consideration with the modal -- that'll work for now, but when we have multiple methods we'd probably want to link to the Create Page (as it is now) where a user can choose import methods.

So if we introduce the modal now in the short term, we may want to change it once we have more methods. Wonder if that'll be less ideal than having it be the page link now since we'll end up there in the medium/longer term. 

@twblack88 curious for your thoughts here since I think there's pros/cons to both